tests/main/interfaces-ros-opt-data/test-snapd-sh/meta/snap.yaml

```diff
@@ -0,0 +1,13 @@
+name: test-snapd-sh
+summary: A no-strings-attached, no-fuss shell for writing tests
+version: 1.0
+
+plugs:
+    opt-ros:  
+        interface: ros-opt-data
+        read: [/opt/ros]
```

Comment:
this attribute is irrelevant no?

Reply:
correct, I removed it. On this regard, I would like to cleanup history or squash all commits together, what is the policy? Will you squash when merging? Should I wait for final review before force-pushing?

Reply:
usually is best to force push only at the end and not during reviews, but yes we can squash merge this when we merge it, we have also a reminder label for that, I added it